content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -38,18 +47,76 @@ and
     +-org.slf4j:jcl-over-slf4j:1.7.26
 ----
 
+This error says that a conflict exists between the versions of `commons-codec` and `jcl-over-slf4j` specified by the plugin versus Jenkins core.
+See link:/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes[Understanding requireUpperBoundDeps failures and fixes] for more information.
+
+In many cases, upper bounds dependencies errors are best resolved by updating the Jenkins core Bill of Materials and by using the Jenkins plugin Bill of Materials.
+
+=== Update the Jenkins Core Bill of Materials (BOM)
+
+Update the parent pom version with the command:
+
+[source,bash]
+----
+$ mvn versions:update-parent
+$ mvn clean verify
+----
 
-What this error is saying is that there is a conflict between the versions of `commons-codec` and `jcl-over-slf4j` specified by the plugin versus Jenkins.
-Without using the BOM, you would need to go and update the dependencies in your plugin to match those required by Jenkins, and then keep changing these as your `jenkins.version` changes.
-With the BOM all you have to do is import the link:https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml[jenkins-bom] for the version of Jenkins you are building against, and the versions of these and other common dependencies will be matched to that version of Jenkins.
+See the video below for a step by step guide to updating the parent pom.
 
-To use the Jenkins Core BOM in your plugin, just use plugin-pom (4.0 or later).
+.Update the parent pom
+video::Fev8KfFsPZE[youtube, width=852, height=480,start=807]
 
-== Jenkins Plugin BOM
+== Jenkins plugin BOM
 
-See the link:https://github.com/jenkinsci/bom[jenkinsci/bom] repository
+The plugin bill of materials (BOM) defines the versions of various plugins used by specific Jenkins versions.
+The plugin bill of materials includes many commonly used plugins.
+It simplifies dependency management for plugins that depend on other plugins.
+
+A plugin that tests its Pipeline implementation will typically depend on several Pipeline plugins.
+The plugin BOM provides Pipeline plugin version numbers so that the developer does not need to maintain the independent version numbers of the Pipeline plugins,
```

Comment:
```suggestion
The plugin BOM provides Pipeline plugin version numbers so that the developer does not need to maintain the independent version numbers of the Pipeline plugins.
```